### PR TITLE
update about page DEI and community sections (fix #15301)

### LINKED ISSUE #15301: Refresh: About - design tweaks
[Details are in the Figma file.](https://www.figma.com/design/umfT9uNYJwAjIwgz360BdB/MVP-3.0-Mozilla.org?node-id=1-28499&t=VaL6cYyET7hTTRRo-1)

- Switch the DEI report block to green
- Update formatting in community block

### PR DESCRIPTION
**Pull request by wen-2018**
## One-line summary

This PR updates about page DEI and community sections to match the design updates.

## Issue / Bugzilla link

https://github.com/mozilla/bedrock/issues/15301

## Testing

- Turn on switch M24_WEBSITE_REFRESH

### CODE REVIEW

File: bedrock/mozorg/templates/mozorg/about/includes/m24/idea.html [hunk 1]
```diff
@@ -4,14 +4,13 @@
...
   <section class="m24-c-showcase">
-    <h2 class="m24-c-showcase-title">IDEA ipsum dolor sit amet</h2>
-    <p class="m24-c-showcase-subtitle">Suspendisse commodo sed erat in pellentesque. In sit amet tortor porttitor, gravida mauris tempor, pharetra erat. Nam vel justo urna.</p>
+    <h2 class="m24-c-showcase-title">Building a fairer future</h2>
+    <p class="m24-c-showcase-subtitle">See how we’re fostering a diverse, inclusive and accessible online environment for all.</p>
     <div class="m24-c-showcase-media">
       <img src="{{ static('img/m24/todo-1-1.svg') }}" alt="" height="500" width="500">
     </div>
-    <p class="m24-c-showcase-cta"><a href="{{ url('mozorg.impact-report.index') }}" class="m24-c-cta" data-cta-text="IDEA">Read the report</a></p>
+    <p class="m24-c-showcase-cta"><a href="{{ url('mozorg.impact-report.index') }}" class="m24-c-cta" data-cta-text="IDEA">See more</a></p>
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/about/includes/m24/idea.html:14]: **Question:** Do we also need to update `data-cta-text` here, or is `IDEA` still a relevant value?

File: bedrock/mozorg/templates/mozorg/about/includes/m24/leadership.html [hunk 2]
```diff
@@ -4,15 +4,21 @@
...
+        <p class="m24-c-showcase-subtitle">Mozilla isn’t just an organization—it’s a global community of passionate volunteers, fellows and contributors who have been building, protecting and shaping the internet with us since 1998.</p>
+        <p class="m24-c-showcase-subtitle">From writing code and spotting bugs to advocating for privacy and keeping the internet open for everyone, they’re the backbone of everything we do, and we couldn’t do this without them. And guess what? You can join the team too!</p>
+      </div>
+    </div>
+
     <div class="m24-c-showcase-media">
       <img src="{{ static('img/m24/todo-1-1.svg') }}" alt="" height="500" width="500">
     </div>
-    <p class="m24-c-showcase-cta"><a href="{{ url('mozorg.about.leadership.index') }}" class="m24-c-cta" data-cta-text="Leadership">Leadership</a></p>
+    <p class="m24-c-showcase-cta"><a href="{{ url('mozorg.about.leadership.index') }}" class="m24-c-cta" data-cta-text="Leadership">Join the movement</a></p>
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/about/includes/m24/leadership.html:20]: **Issue/Suggestion:* the link needs updating here to point to `/contribute/`?

```suggestion
    <p class="m24-c-showcase-cta"><a href="{{ url('mozorg.contribute') }}" class="m24-c-cta" data-cta-text="Join the movement">Join the movement</a></p>
```

Also, should we rename this include from `leadership.html` to `community.html`?

File: bedrock/mozorg/templates/mozorg/about/includes/m24/idea.html (hunk 1 above)
Comment (alexgibson) [reply bedrock/mozorg/templates/mozorg/about/includes/m24/idea.html:14]: Question: Also, is `idea.html` still the most suitable filename?

**Review by alexgibson — CHANGES_REQUESTED** [review]
Updates look good, just a couple of small nits / suggestions 👍

File: bedrock/mozorg/templates/mozorg/about/includes/m24/idea.html (hunk 1 above)
Comment (wen-2018) [reply bedrock/mozorg/templates/mozorg/about/includes/m24/idea.html]: I have update to use `impact` here, but I am open to better options

File: bedrock/mozorg/templates/mozorg/about/includes/m24/leadership.html (hunk 2 above)
Comment (wen-2018) [reply bedrock/mozorg/templates/mozorg/about/includes/m24/leadership.html]: Updated to use `community` here as file name and the link

**Pull request merged**